# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLLibraryEGL.h
# Commit: 204b899f436d
# Full Hash: 204b899f436d99f0eae759af3ccfcbaabfbbb428
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 3f5baab34b1f -r 204b899f436d gfx/gl/GLLibraryEGL.h
--- a/gfx/gl/GLLibraryEGL.h	Thu Jun 11 06:36:55 2020 +0000
+++ b/gfx/gl/GLLibraryEGL.h	Thu Jun 11 06:37:35 2020 +0000
@@ -385,21 +385,21 @@
     return mIsWARP;
   }
 
-  bool HasKHRImageBase() {
+  bool HasKHRImageBase() const {
     return IsExtensionSupported(KHR_image) ||
            IsExtensionSupported(KHR_image_base);
   }
 
-  bool HasKHRImagePixmap() {
+  bool HasKHRImagePixmap() const {
     return IsExtensionSupported(KHR_image) ||
            IsExtensionSupported(KHR_image_pixmap);
   }
 
-  bool HasKHRImageTexture2D() {
+  bool HasKHRImageTexture2D() const {
     return IsExtensionSupported(KHR_gl_texture_2D_image);
   }
 
-  bool HasANGLESurfaceD3DTexture2DShareHandle() {
+  bool HasANGLESurfaceD3DTexture2DShareHandle() const {
     return IsExtensionSupported(ANGLE_surface_d3d_texture_2d_share_handle);
   }
 